Make README slightly better (#136)
LabsDevelopment/Octobot: docs/README.md

* Banning, muting, kicking, etc.
* Reminding you about something if you wish
* Reminding everyone about that new event you made

> * Renaming those annoying hoisted members

Reviewer: ```suggestion
* Renaming those annoying self-hoisting members
```

* Log everything from joining the server to deleting messages
* Listen to music!

*...a-a-and more!*

> [//]: # (if you are reading this, message @mctaylors that he needs to return wiki back)

Reviewer: ```suggestion
[//]: # (if you are reading this, message @mctaylors and ask him to bring back the wiki)
```
